test/unit/test_http.py

```diff
@@ -222,6 +240,18 @@ def test_failed_download(http_client):
 		with pytest.raises(ipfshttpclient.exceptions.StatusError):
 			http_client.download("/fail")
 
+def test_download_timeout(self):
```

Comment:
Drop the `self` here to make the testing work (this is not inside a class, so there is no `self` to refer to here)

Reply:
Oops.. XD